Marlin/src/gcode/calibrate/G34_M422.cpp

```diff
@@ -73,8 +74,13 @@ void GcodeSuite::G34() {
 
   do { // break out on error
 
-    #if NUM_Z_STEPPER_DRIVERS >= 4
-      SERIAL_ECHOLNPGM("Alignment not supported for over 3 steppers");
+    #if NUM_Z_STEPPER_DRIVERS > 4
+      SERIAL_ECHOLNPGM("Alignment not supported for over 4 steppers");
+      break;
+    #endif
+
+    #if NUM_Z_STEPPER_DRIVERS == 4
+      SERIAL_ECHOLNPGM("Alignment for 4 steppers is Experimental!");
       break;
```

Comment:
```suggestion
    #if NUM_Z_STEPPER_DRIVERS == 4
      SERIAL_ECHOLNPGM("Alignment for 4 steppers is Experimental!");
```

I don't think you want to break out here.